### PR TITLE
Admin Instructions Page

### LINKED ISSUE #346: Make instruction page
~1 hour
Make instruction page for volunteers

### PR DESCRIPTION
**Pull request by camwlsn**
App.js: Navigation route to emp/intructions
EmpBanner.js: Added button to emp banner which navigates to emp instructions
EmpNav.js: Removed comments
Instrcutions.js: Instructions page

close #346

### CODE REVIEW

File: MERN Scripts/mern-emsystem/src/components/Empbanner.js
Comment (SpeiserK) [inline MERN Scripts/mern-emsystem/src/components/Empbanner.js:1]: Nice spot for the button

**Review by SpeiserK — APPROVED** [review]
Nice work adding this info!

File: MERN Scripts/mern-emsystem/src/pages/Instructions.js
Comment (SpeiserK) [inline MERN Scripts/mern-emsystem/src/pages/Instructions.js:1]: Well versed instruction page gg

File: MERN Scripts/mern-emsystem/src/components/Empnav.js
Comment (SpeiserK) [inline MERN Scripts/mern-emsystem/src/components/Empnav.js:1]: This guy don't miss...

**Review by zachpren0 — APPROVED** [review]
this is bussin

**Comment by amurph24** [comment]
Nice work!

**Pull request merged**